### PR TITLE
Specifiy family = 6 when prefix is set

### PR DESCRIPTION
**Pull request by lukebarnard1**
Otherwise, it is assumed that the IRC server ONLY does IPv6

**NOTE: This depends on some uncommitted node-irc code: https://github.com/matrix-org/node-irc/pull/11**

### CODE REVIEW

File: lib/irc/ConnectionInstance.js [hunk 1]
```diff
@@ -244,7 +244,8 @@ ConnectionInstance.create = function(server, opts, onCreatedCallback) {
         port: server.getPort(),
         secure: server.useSsl(),
         selfSigned: server.useSslSelfSigned(),
-        retryCount: 0
+        retryCount: 0,
+        family: server.getIpv6Prefix() === undefined ? 4 : 6,
```
Comment (Kegsay) [inline lib/irc/ConnectionInstance.js]:
```js
family: server.getIpv6Prefix() ? 6 : 4,
```
looks clearer to me.

**Comment by Kegsay** [comment]
> Otherwise, it is assumed that the IRC server ONLY does IPv6

OOI, why is this a problem?

**Comment by lukebarnard1** [comment]
Because, with freenode and OFTC, the servers allow both, instead of having a domain for IPv6 only, like Snoonet

**Comment by Kegsay** [comment]
Oh? I'd have thought specifying a v6 source address is enough for the IRCd to say "hey look some random v6 address is talking to me, I'll use v6"? Orrrr?

**Comment by lukebarnard1** [comment]
It's the DNS lookup on the client that will happen to find the A result before the AAAA result

**Comment by Kegsay** [comment]
Ohhhhh *that*. I'm sure I bumped into this...

**Comment by Kegsay** [comment]
LGTM! (Please close #60 then too!)

**Pull request merged**